AdminWebsite/AdminWebsite/Controllers/HearingsController.cs

```diff
@@ -184,15 +184,15 @@ public async Task<ActionResult<HearingDetailsResponse>> EditHearing(Guid hearing
                 return BadRequest(ModelState);
             }
 
-            HearingDetailsResponse hearing;
+            HearingDetailsResponse originalHearing;
             try
             {
-                hearing = await _bookingsApiClient.GetHearingDetailsByIdAsync(hearingId);
+                originalHearing = await _bookingsApiClient.GetHearingDetailsByIdAsync(hearingId);
             }
             catch (BookingsApiException e)
             {
                 _logger.LogError(e,
-                    "Failed to get hearing {hearing}. Status Code {StatusCode} - Message {Message}",
+                    "Failed to get hearing {Hearing}. Status Code {StatusCode} - Message {Message}",
```

Comment:
Where is Hearing declared? as it would appear this should be originalHearing..

Reply:
This is the property name we want to give when we look in app insights

Reply:
all good